src/app/Trade.ts

```diff
@@ -0,0 +1,70 @@
+import { AnimalNames } from '../Enums/AnimalNamesEnum';
+import { ConvertToAnimalObject } from './utils/ConvertToAnimalObject';
+import _ from 'lodash';
```

Comment:
There is only multiply used here from lodash, so maybe just { multiply } ?

Reply:
right. forgot to change it. added to refactor list :)

Reply:
fixed